Mage.Server/src/main/java/mage/server/http/GamesController.java

```diff
@@ -0,0 +1,132 @@
+package mage.server.http;
+
+import com.google.gson.GsonBuilder;
+import mage.constants.ManaType;
+import mage.constants.PlayerAction;
+import mage.server.User;
+import mage.server.game.GameManager;
+import mage.server.http.util.JwtAuthHelper;
+import mage.view.GameView;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(value="/games")
+public class GamesController {
+    @RequestMapping(value = "/{gameId}/players/{playerId}", method = RequestMethod.GET)
```

Comment:
So are these how routes are managed?

Reply:
The [spring](https://spring.io/) framework keys off of these annotations and sets up the routing based on the values given.

Reply:
> So are these how routes are managed?

![magic - imgur](https://user-images.githubusercontent.com/149671/37069875-492c874c-2183-11e8-8838-bbdc5df07286.gif)